mygithub/src/components/github/Profile.js

```diff
@@ -1,58 +1,59 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import RepoList from './RepoList.js';
-class Profile extends Component{
-	
-	render(){
-// Nesting profile component in App component
-		return(
+class Profile extends Component {
```

Comment:
This component could benefit from proptypes for better type checking.  It's recommended to add proptypes for `userData` and `userRepos` to ensure data consistency and catch potential errors early on.